src/client_side.cc

```diff
@@ -3050,7 +3057,8 @@ ConnStateData::switchToHttps(HttpRequest *request, Ssl::BumpMode bumpServerMode)
 {
     assert(!switchedToHttps_);
 
-    sslConnectHostOrIp = request->url.host();
+    sslConnectHostOrIp =  request->url.hostOrIp();
```

Comment:
This is resulted to other 4-5  more unrelated changes. I did not used a new class, just renamed the sslConnectHostOrIp to tlsConnectHostOrIp.

Resolved at commit e01be963

Reply:
Please feel free to undo https://github.com/squid-cache/squid/commit/e01be9639713d226c93ac6ee77b0ca5c407fb3c9 if you believe that renaming causes too many otherwise unnecessary changes -- I did start my change request with an `If...` precondition that can be false.

If you keep the renamed member, then please fix `HERE` and also `s/SSL/TLS/` in the renamed data member description, to improve symmetry with the sister data member.

Reply:
addressed at 7c79ddfb